.storybook/Container.js

```diff
@@ -5,18 +5,6 @@ export default class Container extends Component {
   render() {
     const { story } = this.props;
 
-    return (
-      <div
-        className="storybook-container"
-        style={{
-          padding: '3rem',
```

Comment:
The stories that are larger than the viewport height are put right on the top/bottom edge.  Left this PR, right production:

![image](https://user-images.githubusercontent.com/3360588/70924815-cb7f8c00-1fef-11ea-8817-a5e5556cc7fa.png)

Would like to keep the breathing room via the padding if possible.

Reply:
Re-added the padding. Thanks!